README.md

```diff
@@ -1,34 +1,18 @@
 # linter-htmlhint
 
-This linter plugin for [Linter](https://github.com/AtomLinter/Linter) provides
-an interface to [htmlhint](https://github.com/yaniswang/HTMLHint). It will be
+A plugin for [Atom Linter](https://github.com/AtomLinter/Linter) providing an interface to [HTMLHint](https://github.com/yaniswang/HTMLHint). It will be
 used with files that have the syntax.
 
 ## Installation
 
-Linter package must be installed in order to use this plugin. If Linter is not
```

Comment:
I would leave in some indication that Linter will be automatically installed for the user.